lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -653,3 +660,103 @@ def create_follow_up_encounter(record: REDCapRecord, patient_reference: dict, si
     )
 
     return create_entry_and_reference(encounter_resource, "Encounter")
+
+
+def create_follow_up_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for the follow-up
+    encounter.
+
+    Note: `fu_which_activites` and `fu_missed_activites` are misspelled on
```

Comment:
Just wanted to say I giggle every time I see the prefix on these fields is `fu`.